src/main/scala/nl/knaw/dans/easy/fedora2vault/EasyFedora2vaultApp.scala

```diff
@@ -18,23 +18,36 @@ package nl.knaw.dans.easy.fedora2vault
 import java.io.InputStream
 import java.nio.file.Paths
 
-import better.files.File
+import better.files.{ File, StringExtensions }
 import com.yourmediashelf.fedora.client.FedoraClient
 import javax.naming.ldap.InitialLdapContext
 import nl.knaw.dans.bag.v0.DansV0Bag
 import nl.knaw.dans.easy.fedora2vault.Command.FeedBackMessage
 import nl.knaw.dans.easy.fedora2vault.FoXml.{ getEmd, _ }
-import better.files.StringExtensions
+import nl.knaw.dans.lib.error._
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 
-import scala.util.{ Success, Try }
+import scala.util.{ Failure, Success, Try }
 import scala.xml.{ Elem, Node, XML }
 
 class EasyFedora2vaultApp(configuration: Configuration) extends DebugEnhancedLogging {
   lazy val fedoraProvider: FedoraProvider = new FedoraProvider(new FedoraClient(configuration.fedoraCredentials))
   lazy val ldapContext: InitialLdapContext = new InitialLdapContext(configuration.ldapEnv, null)
   private lazy val ldap = new Ldap(ldapContext)
 
+  def simpleTransForms(input: File, outputDir: File): Try[FeedBackMessage] = {
+    input.lineIterator.map(datasetId => {
+      val subDir = outputDir / datasetId.replaceAll("[^a-zA-Z0-9]+", "-")
+      simpleTransform(datasetId, subDir)
+        .doIfSuccess(println)
```

Comment:
@linda should decide whether she wants to follow progress as with info logging in #2 
https://github.com/DANS-KNAW/easy-fedora2vault/blob/d216bad51781fa25a94075c9cb311d49e8219844/src/main/scala/nl/knaw/dans/easy/fedora2vault/EasyFedora2vaultApp.scala#L63
https://github.com/DANS-KNAW/easy-fedora2vault/blob/d216bad51781fa25a94075c9cb311d49e8219844/src/main/scala/nl/knaw/dans/easy/fedora2vault/EasyFedora2vaultApp.scala#L113

Speaking of that:
* it might de interesting to add the number/size of files. If not causing to much overhead. To be implemented with [EASY-2678](https://drivenbydata.atlassian.net/browse/EASY-2678): file metadata
* Also having doubts fedora returns large number of files all at once with https://github.com/DANS-KNAW/easy-fedora2vault/blob/fd5ce5215dcd2b2a3eea379b7f87243f747fb319/src/main/scala/nl/knaw/dans/easy/fedora2vault/FedoraProvider.scala#L34-L35 after all the [online form](http://deasy.dans.knaw.nl:8080/fedora/risearch) lets you choose the returned number.